packages/jumpstarter-driver-flashers/oci_bundles/ti_j784s4xevm/build_flasher.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+dnf install --setopt=install_weak_deps=false -y git make gcc gcc-c++ which file diffutils wget cpio rsync bc zip patch perl tar qemu-system-aarch64 qemu-img unzboot uboot-tools kmod
+
+git clone --depth 1 --branch 2025.05-rc2  https://github.com/buildroot/buildroot /buildroot
+
+./add_kernel.sh
+cp -R overlay /buildroot
+cp renesas_s4_defconfig  /buildroot/configs/
+( cd /buildroot; make renesas_s4_defconfig && make )
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)